src/nbsap/templates/_bits/account_menu.html

```diff
@@ -19,11 +19,13 @@
       </li>
     {% endfor %}
     {% if user.is_authenticated %}
-      <li>
-        <a href="{% if NAT_STRATEGY %}{% url 'list_national_objectives' %}{% else %}{% url 'list_eu_targets' %}{% endif %}">
-          {% trans 'Dashboard' %}
-        </a>
-      </li>
+      {% if user.is_superuser %}
```

Comment:
You can check only once if the current user is superuser (merge with condition on line 29).